tests/test_land_vision.py

```diff
@@ -19,3 +25,36 @@
 #     print ('ret', json.dumps(ret['tile'], indent = 2))
 
 #     # _land_vision.seeLand(imageUrl)
+
+def test_urban_tree_detection_inference():
```

Comment:
Great job adding a test!

Reply:
thanks. I also changed it a bit to make it resemble more the content of inference.py.